014-quarkus-panache-with-transactions-xa/src/main/resources/application.properties

```diff
@@ -7,7 +7,13 @@ quarkus.datasource.username=sarah
 quarkus.datasource.password=connor
 quarkus.datasource.jdbc.url=jdbc:mysql://localhost:3306/db?currentSchema=test
 quarkus.datasource.jdbc.transactions=enabled
-quarkus.datasource.jdbc.max-size=1
```

Comment:
Nack: Setting `max-size` to 1 was done on purpose to ensure no more than 1 connection is created. Indeed, thanks to this setting, we have been able to spot issues on Quarkus upstream.